### PR TITLE
DC-09 Contact Section

### PR DESCRIPTION
**Pull request by mayurdawnge45**
Hi @akash-dhande I have updated this ticket work please merge this ticket
https://dhandecreative.atlassian.net/browse/DC-9

Thanks

### CODE REVIEW

**Comment by akash-dhande** [comment]
Branch has wrong base, following old commints.

**Review by akash-dhande — CHANGES_REQUESTED** [review]
wrong base, missing PR details

File: index.html [hunk 1]
```diff
@@ -50,7 +52,11 @@ <h2>More Details <br>
                 </h2>
             </div>
             <div class="right-banner">
+<<<<<<< HEAD
+                <img src="assets/images/banner.png" alt="">
+=======
                 <img src="assets/images/banner-logo.png" alt="">
+>>>>>>> master
```
Comment (akash-dhande) [inline index.html]: Hi @mayurdawnge45, you missed conflict marks.

**Review by akash-dhande — CHANGES_REQUESTED** [review]

**Comment by akash-dhande** [comment]
Also please update PR title and description.

**Comment by mayurdawnge45** [comment]
Hi @akash-dhande I have Updated some work related to this ticket https://dhandecreative.atlassian.net/browse/DC-9
please merged this ticket

Thanks

**Review by akash-dhande — APPROVED** [review]

**Pull request merged**